src/lib/theme/default.ts

```diff
@@ -240,7 +240,7 @@ export default {
     },
   },
   carousel: {
-    base: 'relative h-full w-full min-h-56 sm:min-h-64 xl:min-h-80 2xl:min-h-96',
+    base: 'relative h-full w-full min-h-[3.5rem] sm:min-h-[4rem] xl:min-h-[5rem] 2xl:min-h-[6rem]',
```

Comment:
Thank you! We should try to match with Flowbite itself and use the values provided by the original component here: https://flowbite.com/docs/components/carousel/

For example, the `2xl` breakpoint is set to `h-96` which translates to `24rem`